buildkite/bazelci.py

```diff
@@ -1746,7 +1746,7 @@ def print_bazel_downstream_pipeline(
                         fetch_bazelcipy_command(),
                         fetch_aggregate_incompatible_flags_test_result_command(),
                         python_binary()
-                        + " aggregate_incompatible_flags_test_result.py --build_number=%s | buildkite-agent pipeline upload"
```

Comment:
Can we really delete the upload here?

Reply:
Yes, we are not print any job config content to be uploaded anymore.